daycare/test_daycare.py

```diff
@@ -61,3 +64,19 @@ def test_daycare_user_creation_with_same_user_name(self, create_daycare_user):
             DayCare.create(username='testUser01', email="valid@gmail.com", password='pass123', name='daycare',
                            description='new description', price_per_day=10, capacity=50,
                            area='north', city='haifa', address='new address')
+
+    def test_daycare_redirected_homepage_after_login(self, client, create_daycare_user):
+        client.force_login(user=create_daycare_user.user)
+        response = client.get('/')
+        assert response.status_code == 302
+        assert response['Location'] == '/homepage/'
+
+    def test_relevant_daycare_homepage_with_static_data(self, client, create_daycare_user):
+        dayCare = random.choice([daycare for daycare in DayCare.objects.all() if 'static' in daycare.user.username])
+        client.force_login(user=dayCare.user)
+
+        response = client.get('/homepage/')
+
+        assert response.context['daycare'] == dayCare
+        assert set(response.context['images']) == set(Image.get_images_by_daycare_id(daycare_id=dayCare.id))
```

Comment:
would improve readability to assign both sides of the assertion to variable with clear names before the assetion.

Reply:
good one! added meaningful names for homepage variables